lxd/util/net_test.go

```diff
@@ -68,8 +68,8 @@ func TestIsAddressCovered(t *testing.T) {
 		{":8443", "0.0.0.0:8443", false},
 		{"127.0.0.1:8443", "[::]:8443", true},
 		{"[::1]:8443", "[::]:8443", true},
-		{"[::1]:8443", ":8443", true},
-		{":8443", "[::]:8443", true},
+		{"[::1]:8443", ":8443", false},
+		{":8443", "[::]:8443", false},
```

Comment:
First case is IPv6 local (`[::1]`) being covered by a wildcard listener on port `8443`. This is covered.
Second case is comparing two effectively identical things, the wildcard listener on port `8443` with the IPv6 wildcard listener also on port 8443, this should similarly be covered.

There are effectively 3 wildcard syntaxes:
 - 0.0.0.0:8443 (all IPv4 addresses)
 - [::]:8443 (all IPv6 addresses, which ends up including all IPv4 addresses too on most systems)
 - :8443 (all IPv4 and IPv6 addresses, this is a Go specific syntax thing, not standard)

`:8443` covers `0.0.0.0:8443` and `[::]:8443`
`[::]:8443` covers `0.0.0.0:8443` and `:8443`
`0.0.0.0:8443` doesn't cover either of the other (as it doesn't do IPv6)